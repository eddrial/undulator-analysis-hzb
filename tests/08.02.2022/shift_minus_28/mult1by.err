 error report for the data set mult1by.dvm         
--------------------------------------------
      
         127   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1591 data points 
        out of totally   2558 data points
 the derivative at  0.40187E+03 and  0.42427E+03 is zero 
 the derivative at  0.42427E+03 and  0.20844E+04 is zero 
 the derivative at  0.20844E+04 and  0.21064E+04 is zero 
 the rms value of the residuals is  0.43289E-02
 the maximum residual is  0.37515E-01
 the rms value of the lambda/2-filtered data is  0.16781E-01
 the maximum value of the lambda/2-filtered data is  0.10302E+00
