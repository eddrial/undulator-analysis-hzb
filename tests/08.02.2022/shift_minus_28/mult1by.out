mult1by.dvm 
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1 -2.5256056E-02
   2  5.7894381E-04
   3  5.8738363E-04
   4  6.2985018E-05
   5  3.2202941E-05
   6  5.6039185E+01
   7  1.0470632E+03
   8  1.0472317E+03
   9  1.0452012E+03
  10 -1.1783274E+00
  11 -2.3650699E+00
 number of iterations:   12
     sumsq:  2.98140E-02
       rms:  4.32887E-03  dimension of magnetic field
       rms:  1.71392E+01  percent of b_eff
   rms_ext:  0.00000E+00  percent of b_eff
 B_eff =   2.5257E-02
 B_max =  -2.4355E-02
 K_eff =    0.132
 lamda_1 (nm) =      2.56245

        eigenvectors in parameter space

 error                         direction
     4.74,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
     3.12,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
     0.79,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     0.47,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.06,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.01,  0.00  0.00  0.00  0.00  0.00  1.00  0.05  0.00  0.00  0.00  0.00
     0.00,  0.77  0.59  0.09 -0.04 -0.22  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.50 -0.40  0.09 -0.25  0.72  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.37 -0.45 -0.45  0.63 -0.23  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.07 -0.28  0.88  0.31 -0.21  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.16  0.45  0.06  0.67  0.57  0.00  0.00  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:  -0.07249 Tmm
 kick_2:   0.07249 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.81104E+01
      
 the truncated data set has only   1591 data points 
      
 the rms value of the residuals is  0.43289E-02
 the maximum residual is  0.37515E-01
      
 the rms value of the lambda/2-filtered data is  0.16781E-01
 the maximum value of the lambda/2-filtered data is  0.10302E+00
      
 the rms phase error (all data) is    2.651 deg.
 the maximum phase error (all data) is    8.584 deg.
      
 the Walker-rms phase error (all data) is    2.650 deg.
 the Walker-maximum phase error (all data) is    8.568 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             2.651120    
 local phase error             1.020275    
 slowly varying phase error    2.433167    
 oscillatory phase error      0.2591799    
 -----------------------------------------
